aspnetcore/security/authentication/mfa.md

```diff
@@ -0,0 +1,562 @@
+---
+title: Multi-factor authentication
+author: damienbod
+description: Multi-factor authentication
+monikerRange: '>= aspnetcore-3.1'
+ms.author: rick-anderson
+ms.custom: mvc
+ms.date: 02/25/2020
+no-loc: [Identity]
+uid: security/authentication/mfa
+---
+# Multi-factor authentication
+
+By [Damien Bowden](https://github.com/damienbod)
+
+Multi-factor authentication (MFA) is a process where a user is requested during a sign-in event for additional forms of identification. This prompt could be to enter a code on their cellphone, use a FIDO2 key or to provide a fingerprint scan. When you require a second form of authentication, security is increased as this additional factor isn't something that's easy for an attacker to obtain or duplicate.
+
+This article covers the following:
+
+* What is MFA and what MFA flows are recommended
+* Configure MFA for administration pages using ASP.NET Core Identity
+* Send MFA signin requirement to OpenID Connect server 
+* Force ASP.NET Core OpenID Connect client to require MFA
+
+## MFA, 2FA
+
+MFA requires at least two or more types of proof for an identity like something you know, possess, or inherit for the user to authenticate.
+
+Two-factor authentication (2FA) is like a subset of MFA, but the difference being that MFA can require two or more factors to prove the identity.
+
+### MFA TOTP (Time-based One-time Password Algorithm)
+
+MFA using TOTP is a supported implementation using ASP.NET Core Identity. This can be used together with the following apps:
+
+* Microsoft Authenticator App
+* Google Authenticator App
+
+See the following link for implementation details:
+
+[Enable QR Code generation for TOTP authenticator apps in ASP.NET Core](xref:security/authentication/identity-enable-qrcodes)
+
+### MFA FIDO2 or passwordless
+
+FIDO2 is:
+
+* The most secure way of achieving MFA.
+* The only MFA flow that protects against phishing attacks.
+
+At present, ASP.NET Core doesn't support FIDO2 directly. FIDO2 can be used for MFA or passwordless flows.
+
+Azure Active Directory provides support for FIDO2 and passwordless flows. For more information, see [Passwordless authentication options for Azure Active Directory](/azure/active-directory/authentication/concept-authentication-passwordless).
+
+You can implement ASP.NET Core with FIDO2 by using the following OSS FIDO2 implementation:
```

Comment:
Given this is a security topic should we link to things we haven't examined for security?

Reply:
removed the OSS link